conf/registry-inmemory-example.yaml

```diff
@@ -60,6 +60,11 @@ swagger:
 #enable CORS, may want to disable in production
 enableCors: false
 
+#Set below 3 properties if server needs a proxy to connect. Useful to download mysql jar
```

Comment:
nit : i dont think we should reference mysql in this file

Reply:
Addressed